apps/docs/content/guides/platform/manage-your-usage/disk-size.mdx

```diff
@@ -0,0 +1,110 @@
+---
+id: 'manage-usage-disk-size'
+title: 'Manage Disk size usage'
+---
+
+## What you are charged for
+Each Supabase project has a dedicated [disk](/docs/guides/platform/compute-and-disk#disk). You are charged for the provisioned disk size.
+
+<Admonition type="note">
+  Disk size is not relevant for the Free Plan. Instead Free Plan customers are limited by [Database size](/docs/guides/platform/database-size).
+</Admonition>
+
+## How charges are calculated
+Disk size is charged by Gigabyte-Hours (GB-Hrs). 1 GB-Hrs represents 1 GB being provisioned for 1 hour.
+For example, having 10 GB provisioned for 5 hours results in 50 GB-Hrs (10 GB × 5 hours).
+
+### Disk size usage on your invoice
+Disk size usage is shown as "Disk Size GB-Hrs" on your invoice.
+
+## Pricing
+Pricing depends on the [disk type](/docs/guides/platform/compute-and-disk#disk-types), with type gp3 being the default.
+
+### General purpose disks (gp3)
+
+1 GB-Hrs is $0.000171 ($0.125 per GB per month).
+Each project gets provisioned with an 8 GB disk for free. You are only charged for the disk size exceeding these 8 GB.
```

Comment:
Didn't use the word "quota" here. Think it might be misleading. To me "quota" would imply that you start from zero.